### ISSUE TITLE
Static image in Edit Event

### ISSUE THREAD

**Issue by ChetanaKoulagi** [LetsTagOn/LetsTagOnThoughtClan#23]
When we click on a Program, it shows the related events. In that, the images shown in related events are static images and not taken from the Event.

**Comment by ChetanaKoulagi** [comment]
Where are the static images coming from for an event? Are they hardcoded or do they have to be uploaded?

**Comment by Viswa-Vinod** [comment]
I checked the logic. The page looks for the existence of an event-related image. If it does not find an event-related image, then it uploads a static image. Do you want any changes to this logic?

**Comment by ChetanaKoulagi** [comment]
Even if I have uploaded an event-related image, it shows these static images.

**Comment by Viswa-Vinod** [comment]
You are right. I have fixed that. Please check after I push all my changes to my branch. I will let you know when I do finish pushing.

**Comment by ChetanaKoulagi** [comment]
I see the event related images now in Event, but other details are missing. Is it because of fix of this issue? Can you please check.

**Comment by Viswa-Vinod** [comment]
The logic currently is such that only if postal code and street name are provided, the details will start to show. Providing city name or country is insufficient. 
Also, as of now no contact person name has been provided for the related events. 
Do you want me to change the logic or do you want to provide all the data as per current logic? Please advise.
